src/main/java/ch/kalunight/zoe/repositories/ConfigRepository.java

```diff
@@ -0,0 +1,297 @@
+package ch.kalunight.zoe.repositories;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import ch.kalunight.zoe.Zoe;
+import ch.kalunight.zoe.model.config.ServerConfiguration;
+import ch.kalunight.zoe.model.config.option.CleanChannelOption;
+import ch.kalunight.zoe.model.config.option.CleanChannelOption.CleanChannelOptionInfo;
+import ch.kalunight.zoe.model.config.option.GameInfoCardOption;
+import net.dv8tion.jda.api.entities.Role;
+import net.dv8tion.jda.api.entities.TextChannel;
+import net.rithms.riot.constant.Platform;
+import ch.kalunight.zoe.model.config.option.RegionOption;
+import ch.kalunight.zoe.model.config.option.RoleOption;
+import ch.kalunight.zoe.model.config.option.SelfAddingOption;
+
+public class ConfigRepository {

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'ConfigRepository' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/KaluNight/Zoe-Discord-Bot/pullRequest?prid=4739924)